spotinst_sdk2/clients/ocean/__init__.py

```diff
@@ -2128,10 +2129,13 @@ def create_virtual_node_group(self, vng: gcp_ocean.VirtualNodeGroup):
 
         body_json = json.dumps(formatted_missing_dict)
 
-        response = self.send_post(
+        query_params = dict(initialNodes=initial_nodes)
```

Comment:
Do we need to add a 'None' check here?

Reply:
Yes Anurag, This is similar to AWS. The function should work even when that initial_nodes param is not provided